### PR TITLE
Missing pointer-cursor for tag-badge in article view

### PR DESCRIPTION
**Pull request by nunocoracao**
Fixes https://github.com/nunocoracao/blowfish/issues/956

### CODE REVIEW

File: layouts/partials/article-meta/basic.html [hunk 1]
```diff
@@ -47,7 +47,7 @@
 {{ end }}
 
 
-<div class="flex flex-row flex-wrap items-center">
+<div style="cursor: default;" class="flex flex-row flex-wrap items-center">
```
Comment (JVimes) [inline layouts/partials/article-meta/basic.html:50]: @nunocoracao I think this hides the pointer cursor for article date link. Is that on purpose?

Comment (nunocoracao) [reply layouts/partials/article-meta/basic.html:50]: @JVimes yes, did you have any use for it?

Comment (JVimes) [reply layouts/partials/article-meta/basic.html:50]: Without it, I'd assume the site doesn't link article dates.

Comment (nunocoracao) [reply layouts/partials/article-meta/basic.html:50]: Sorry @JVimes I am not following the use-case. Can you point me to an example of what worked and doesn't anymore. Maybe we should open a issue for this also.

Comment (JVimes) [reply layouts/partials/article-meta/basic.html:50]: Never mind, I'm getting my wires crossed with my modifications 🙂

**Pull request merged**